NBitcoin/Transaction.cs

```diff
@@ -1083,7 +1083,8 @@ public enum TransactionOptions : uint
 		Witness = 0x40000000,
 		All = Witness
 	}
-	class Witness
+
+	public class Witness

```

Comment:
Don't use public here, at worst copy the Witness implementation in Stratis code.

Reply:
Or maybe use InternalsVisibleTo("NBitcoin.Altcoins")

Reply:
👍